src/h5web/visualizations/containers/NxImageContainer.tsx

```diff
@@ -62,12 +68,12 @@ function NxImageContainer(props: VisContainerProps): ReactElement {
         onChange={setMapperState}
       />
       <MappedHeatmapVis
-        value={values.signal}
-        valueLabel={signal}
+        value={signal.values}
+        valueLabel={signal.label}
```

Comment:
I am biased as it is called `label` in matplotlib so I find that it is the most appropriate word...

Reply:
Ah hmm weird. The good thing about `title` is that it can stand on its own (as opposed to `label`/`valueLabel`) -- it's clear that it's the plot's title and not an axis label. I don't know, I feel like `title` is clearer, but I trust you.

Reply:
Sorry, I thought we were talking about the `LineVis`. You are right, `title` makes much more sense here for the `HeatmapVis`